app/components/initEndpoint/initEndpointController.js

```diff
@@ -26,7 +26,7 @@ function ($scope, $state, EndpointService, StateManager, EndpointProvider, Notif
   $scope.createLocalEndpoint = function() {
     $('#createResourceSpinner').show();
     var name = 'local';
-    var URL = 'unix:///var/run/docker.sock';
+    var URL = 'npipe:////./pipe/docker_engine';
```

Comment:
I also thought how we get the server OS here. Maybe this should be left empty and hte EndponitService uses the defaults for the correct platform? @deviantony WDYT